File: DotNet/CesiumLanguageWriter/Advanced/CesiumCartographicRadiansListOfListsValuePropertyAdaptor.cs
using System.Collections.Generic;
using JetBrains.Annotations;

namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// Adapts a class that implements <see cref="ICesiumCartographicRadiansListOfListsValuePropertyWriter"/> to implement
    /// <see cref="ICesiumPropertyWriter"/> for a list of lists of <see cref="Cartographic"/> values.
    /// </summary>
    /// <typeparam name="TFrom">The class that implements <see cref="ICesiumCartographicRadiansListOfListsValuePropertyWriter"/> to adapt.</typeparam>
    public class CesiumCartographicRadiansListOfListsValuePropertyAdaptor<TFrom> : CesiumWriterAdaptor<TFrom, IEnumerable<IEnumerable<Cartographic>>>
        where TFrom : class, ICesiumCartographicRadiansListOfListsValuePropertyWriter
    {
        /// <summary>
        /// Initializes a new instance.
        /// </summary>
        /// <param name="parent">The instance to wrap.</param>
        /// <param name="writeValueCallback">The callback to write a list of lists of <see cref="Cartographic"/> values.</param>
        /// <param name="writeDeleteValueCallback">The callback to write an indication that the client should delete existing data.</param>
        public CesiumCartographicRadiansListOfListsValuePropertyAdaptor([NotNull] TFrom parent,
                                                                        [NotNull] CesiumWriterAdaptorWriteCallback<TFrom, IEnumerable<IEnumerable<Cartographic>>> writeValueCallback,
                                                                        [NotNull] CesiumWriterAdaptorWriteDeleteCallback<TFrom> writeDeleteValueCallback)
            : base(parent, writeValueCallback, writeDeleteValueCallback)
        {
        }
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/CesiumCartographicRadiansListOfListsValuePropertyAdaptor.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;
import javax.annotation.Nonnull;

/**
 * Adapts a class that implements {@link ICesiumCartographicRadiansListOfListsValuePropertyWriter} to implement
 {@link ICesiumPropertyWriter} for a list of lists of {@link Cartographic} values.
 * @param <TFrom> The class that implements {@link ICesiumCartographicRadiansListOfListsValuePropertyWriter} to adapt.
 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public class CesiumCartographicRadiansListOfListsValuePropertyAdaptor<TFrom extends ICesiumCartographicRadiansListOfListsValuePropertyWriter> extends
        CesiumWriterAdaptor<TFrom, Iterable<? extends Iterable<Cartographic>>> {
    /**
    * Initializes a new instance.
    * @param parent The instance to wrap.
    * @param writeValueCallback The callback to write a list of lists of {@link Cartographic} values.
    * @param writeDeleteValueCallback The callback to write an indication that the client should delete existing data.
    */
    public CesiumCartographicRadiansListOfListsValuePropertyAdaptor(@Nonnull TFrom parent,
            @Nonnull CesiumWriterAdaptorWriteCallback<TFrom, Iterable<? extends Iterable<Cartographic>>> writeValueCallback,
            @Nonnull CesiumWriterAdaptorWriteDeleteCallback<TFrom> writeDeleteValueCallback) {
        super(parent, writeValueCallback, writeDeleteValueCallback);
    }
}